### DISCUSSION TITLE
MongoDB backend

### DISCUSSION THREAD

**Discussion by fcoury** [ZeroAssumptions/aide-de-camp#18] [Show and tell]
Hi there, I implemented a MongoDB backend for aide-de-camp, take a look:

https://github.com/gistia/aide-de-camp-mongodb

All the tests on sqlite's lib.rs work. 

Would love to get some feedback, thank you!

**Comment by andoriyu** [comment]
Thank you! I will take a look and add it to the readme as an option.

**Reply by andoriyu** [reply]
Looks good. Added to readme.

**Reply by fcoury** [reply]
Thank you @andoriyu
